src/test/java/address/testutil/ContextMenuChoice.java

```diff
@@ -0,0 +1,8 @@
+package address.testutil;
+
+/**
+ *
+ */
+public enum ContextMenuChoice {
+        EDIT, TAG, DELETE, CANCEL;
+}
```

Comment:
yea, in anyways tester still need to modified the existing codes when there are more options. Having small methods rather than the current one big methods help in the readability also.